image/templates/helm/stackrox-secured-cluster/internal/defaults/50-images.yaml.htpl

```diff
@@ -33,13 +31,12 @@ image:
     registry: {{ ._rox.image.registry }}
   collector:
     registry: {{ if or (eq ._rox.image.registry "stackrox.io") (eq ._rox.image.registry "registry.connect.redhat.com") }}collector.stackrox.io{{ else }}{{ ._rox.image.registry }}{{ end }}
+  # TODO: Shouldn't the [<- if not .KubectlOutput >] check be here?
```

Comment:
nit: assuming already planned, but these comments need to be removed prior to merging, the comment is not understood by the 'meta templator' and the `[<- if not .KubectlOutput >]` command is processed (in this case breaking the yaml structure)